## Lesson 4 ##

### Task 1 ###

Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

#### Решение ####

In [4]:
M = (800 + 200)/2
M

300.0

In [5]:
D = (800 - 200)**2 /12
D

30000.0

Ответ: среднее значение 500, дисперсия 30000

### Task 2 ###

О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?

#### Решение ####

In [12]:
a = 0.5

0.2 = (b - a)**2 / 12

sqrt(0.2 * 12) = (b - a)

In [13]:
b = (0.2 * 12)**(1/2) + a
b

2.049193338482967

In [16]:
M = (a+b)/2
M

1.2745966692414834

Ответ: b = 2.05, M = 1.27

### Task 3 ###

Непрерывная случайная величина X распределена нормально и задана плотностью распределения
f(x) = (1 / (4 * sqrt(2 * pi))) * (exp(-((x+2) ** 2) / 32))
Найдите:

а). M(X)

б). D(X)

в). std(X) (среднее квадратичное отклонение)


#### Ответ: ####

a) M(X) = -2

б) D(X) = 16

в) std(X) = 4


### Task 4 ###

Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:

а). больше 182 см

б). больше 190 см

в). от 166 см до 190 см

г). от 166 см до 182 см

д). от 158 см до 190 см

е). не выше 150 см или не ниже 190 см

ё). не выше 150 см или не ниже 198 см

ж). ниже 166 см.

#### Решение ####

##### a) больше 182 см #####

In [17]:
M = 174
S = 8

In [21]:
Z_a = (182 - 174)/8
Z_a

1.0

Определяем вероятность того, что значение меньше 1.0 по таблице Z-распеделения

In [23]:
P_za = 0.8413
P_a = 1 - P_za
P_a

0.15869999999999995

Ответ a): 15.87%

##### б) больше 190 см #####

In [22]:
Z_b = (190 - 174)/8
Z_b

2.0

Определяем вероятность того, что значение меньше 2.0 по таблице Z-распеделения

In [24]:
P_zb = 0.9772
P_b = 1 - P_zb
P_b

0.022800000000000042

Ответ б): 2.28%

##### в) от 166 см до 190 см #####

In [26]:
Z_c = (166 - 174)/8
Z_c

-1.0

Определяем вероятность того, что значение меньше -1.0 по таблице Z-распеделения

In [28]:
P_zc = 0.1587

In [29]:
P_c = P_zb - P_zc
P_c

0.8185

Ответ в): 81.85%

##### г) от 166 см до 182 см #####

In [30]:
P_d = P_za - P_zc
P_d

0.6826000000000001

Ответ г): 68.26%

##### д) от 158 см до 190 см #####

In [31]:
Z_e = (158-174)/8
Z_e

-2.0

Определяем вероятность того, что значение меньше -2.0 по таблице Z-распеделения

In [32]:
P_ze = 0.0228

In [36]:
P_e = P_zb - P_ze
P_e

0.9543999999999999

Ответ д): 95.43%

#### е) не выше 150 см или не ниже 190 см ####

In [37]:
Z_f = (150-174)/8
Z_f

-3.0

Определяем вероятность того, что значение меньше -3.0 по таблице Z-распеделения

In [38]:
P_zf = 0.0014

In [39]:
P_f = P_zf + P_b
P_f

0.02420000000000004

Ответ e): 2.42%

#### ё) не выше 150 см или не ниже 198 см ####

In [40]:
Z_g = (198-174)/8
Z_g

3.0

Определяем вероятность того, что значение меньше 3.0 по таблице Z-распеделения

In [41]:
P_zg = 0.9986

In [42]:
P_g = P_zf + (1 - P_zg)
P_g

0.002799999999999957

Ответ ё): 0.28%

#### ж) ниже 166 см ####

In [43]:
P_i = P_zc
P_i

0.1587

Ответ ж): 15.87%

### Task 5 ###

На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

#### Решение ####

In [44]:
M = 178
D = 25
S = D ** (1/2)
S

5.0

In [45]:
d = (190 - M)/S
d

2.4

Ответ: 2.4